sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/internal/aggregator/DoubleExponentialHistogramBuckets.java

```diff
@@ -249,6 +245,34 @@ int getScaleReduction(long newStart, long newEnd) {
     return scaleReduction;
   }
 
+  private static int getIndexByLogarithm(double value, int scale) {
+    double scaleFactor = Math.scalb(1D / Math.log(2), scale);
+    return (int) Math.floor(Math.log(value) * scaleFactor);
+  }
+
+  private static int getIndexByExponent(double value, int scale) {
+    return Math.getExponent(value) >> -scale;
+  }
+
+  /**
+   * Maps a recorded double value to a bucket index.
+   *
+   * <p>The strategy to retrieve the index is specified in the OpenTelemtry specification:
+   * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/datamodel.md#exponential-buckets
+   *
+   * <p>If the index cannot be represented by a signed 32 bit integer, it is expected the histogram
+   * will be downscaled.
```

Comment:
```suggestion
   * <p>Indexes outside the range of a signed 32-bit integer indicate an incorrect scale.
```

Is this what this line means? Not quite sure what the expectation actually refers to

Reply:
I've removed this line, it isn't really necessary and only tangentially related to this method, so it's causing more confusion than it's worth.